static/src/javascripts/projects/common/modules/commercial/hosted-gallery.js

```diff
@@ -299,6 +301,8 @@ define([
                     bonzo(this.$galleryEl).toggleClass('show-oj', this.index === this.$images.length);
                     bonzo(this.$galleryEl).toggleClass('show-cta', this.index === config.page.ctaIndex + 1);
                 }
+
+                url.pushUrl({}, document.title, config.page.pageName + '#img-' + this.index, true);
```

Comment:
Could we not just use `window.location.hash = 'blahblah'`? This also ads it to the back button - which might be a consideration as if I have gone through 20 images, I have to hit back 20 times before actually going back/